docs/development/contribution/how_to_contribute_code.md

```diff
@@ -104,6 +110,12 @@ bin/zeppelin-daemon start
 
 Server will be run on [http://localhost:8080](http://localhost:8080).
 
+#### Option 3 - IDE
+
+1. Copy the `conf/zeppelin-site.xml.template` to `zeppelin-server/src/main/resources/zeppelin-site.xml` and change the configurations in this file if required
+2. `ZeppelinServer.java` Main class
+
+
```

Comment:
Thanks for the contribution for how to debugging zeppelin-server. Personally I would recommend to use Daemon script, because daemon script would also do some other set up for zeppelin server (like run zeppelin-env.sh), and other approaches (Command Line & IDE) don't do that.

Reply:
In my case, it was enough to run the `ZeppelinServer.java` Main class to test the changes on the fly. It required much less effort from my side to set up the IDE to run the Main class. Though it worth adding this information to the `how_to_contribute` file.